std_msgs/Header header
StereoMeas[] stereo_features
